### PR TITLE
🖱️ selection: re-set selection on change

### LINKED ISSUE #322: browser-selection gets unset when interacting with the export popup
this makes the rendering a bit wiered but does not affect functionality

### PR DESCRIPTION
**Pull request by anuejn**
fixes #322

### CODE REVIEW

**Comment by anuejn** [comment]
this leads to some wiered behaviour (like text in the export dialog being effectively unselectable) but I think thats okay?

**Comment by anuejn** [comment]
alternatively we could just set the selection again, after the export dialog is closed

File: app/src/pages/Editor/Document.tsx [hunk 1]
```diff
@@ -230,6 +237,16 @@ const itemFromNode = (content: TimedDocumentItem[], node: Node, n = 0) => {
   return getItem(content, element);
 };
 function SelectionApply({ documentRef }: { documentRef: RefObject<HTMLDivElement> }): JSX.Element {
+  const [updateCount, setUpdateCount] = useState(0);
+  useEffect(() => {
+    const listener = (e: Event) => {
+      console.log(e);
```
Comment (pajowu) [inline app/src/pages/Editor/Document.tsx:243]: 🖨️

**Review by pajowu — APPROVED** [review]

**Comment by anuejn** [comment]
I am not convinced anymore, that this is the propper solution. Probably I will close this once I come up with something better in the next few days...

**Comment by anuejn** [comment]
this is bork

**Pull request closed without merging**